ocs_ci/ocs/platform_nodes.py

```diff
@@ -432,3 +512,389 @@ def restart_nodes_teardown(self):
         # Start the instances
         if stopped_instances:
             self.aws.start_ec2_instances(instances=stopped_instances, wait=True)
+
+    def attach_nodes_to_cluster(self, node_list):
+        """
+        Attach nodes in the list to the cluster
+
+        Args:
+            node_list (list): of AWSUPINode/AWSIPINode objects
+
+        """
+        if self.deployment_type.lower() == 'upi':
+            self.attach_nodes_to_upi_cluster(node_list)
+
+    def attach_nodes_to_upi_cluster(self, node_list):
+        """
+        Attach node to upi cluster
+
+        Args:
+            node_list (list): of AWSUPINode objects
+
+        """
+        if node_list[0].node_type == 'RHEL':
+            self.attach_rhel_nodes_to_upi_cluster(node_list)
+
+    def attach_rhel_nodes_to_upi_cluster(self, node_list):
```

Comment:
can we call ocs_ci.deployment.aws.AWSUPI.run_ansible_playbook instead?

Reply:
@ebenahar ,
I will remove those functions from deployment.aws and this will be the common place. I have a ticket here https://github.com/red-hat-storage/ocs-ci/issues/1723

Reply:
resolving based on above ticket link.